### PR TITLE
data DEEPRAJKUSHWAHA

### PR DESCRIPTION
**Pull request by DEEPRAJKUSHWAHA**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5238"></a>

### CODE REVIEW

**Review by devvsakib — APPROVED** [review]
LGTM

**Review by SurajPratap10 — APPROVED** [review]
LGTM 

`suggestion:` you can add more links and some sections like tags if you like to.

File: DEEPRAJKUSHWAHA.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "name": "DEEPRAJ KUSHWAHA",
+  "type": "personal",
+  "bio": "I am a android and full stack developer"
```
Comment (kumarsonsoff3) [inline DEEPRAJKUSHWAHA.json:4]: There is a missing comma which is leading json check fail. 

```suggestion
  "bio": "I am a android and full stack developer",
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Your file should be in data directory, to move it using GitHub UI, rename your [file here](https://github.com/DEEPRAJKUSHWAHA/LinkFree/edit/DEEPRAJKUSHWAHA-patch-1/DEEPRAJKUSHWAHA.json) to `data/DEEPRAJKUSHWAHA.json`. 

Change `DEEPRAJKUSHWAHA.json`  - >  `data/DEEPRAJKUSHWAHA.json`

Let us know if you need any help with this. 🛠

image

**Review by samyabrata-maji — APPROVED** [review]

**Comment by SurajPratap10** [comment]
@DEEPRAJKUSHWAHA  
Put `DEEPRAJKUSHWAHA.json` file inside the `data` folder, like this -->  `data/DEEPRAJKUSHWAHA.json`

**Comment by Pradumnasaraf** [comment]
> @DEEPRAJKUSHWAHA Put `DEEPRAJKUSHWAHA.json` file inside the `data` folder, like this --> `data/DEEPRAJKUSHWAHA.json`

Hey, 

Please refrain from requesting for same changes if someone has already requested that. It creates an extra notification for everyone. Thank you for understanding.

**Review by TRohit20 — APPROVED** [review]

**Review by vedprakashnautiyal — APPROVED** [review]

**Comment by SaraJaoude** [comment]
We look forward to seeing your Profile on LinkFree, but as there is no recent activity from you to make the changes suggested I am going to close this pull request.

We hope you will raise a new pull request to create your Profile and when you do, you might want to take a look at the following Docs which will help with the suggested changes the maintainers have proposed:

Quickstart: https://linkfree.eddiehub.io/docs/quickstart

Editing your Profile: https://linkfree.eddiehub.io/docs/how-to-guides/editing

**Pull request closed without merging**